lib/pbench/server/api/resources/users_api.py

```diff
@@ -217,16 +217,13 @@ def post(self):
                 500, message="INTERNAL ERROR",
             )
 
-        # check if the user is already logged in, in case the request has a an authorization header included
+        # Log if the request has an authorization header included and user is still trying to create a new login
         # We would still proceed to login and return a new auth token to the user
-        if user == self.auth.token_auth.current_user():
-            self.logger.warning("User already logged in and trying to re-login")
-            if self.auth.token_auth.get_auth()["token"] == auth_token:
-                # If a user attempts to re-login immediately within a millisecond,
-                # the new auth token will be same as the one present in the header
-                # Therefore we will not fulfill the re-login request
-                self.logger.info("Attempted immediate re-login")
-                abort(403, message="Retry login after some time")
+        if self.auth.token_auth.current_user():
+            self.logger.warning(
+                "User already logged in and trying to re-login. Username: {}",
+                {user.username},
```

Comment:
Do you mean to make `user.username` into a `dict`?  Seems like you'd just want to use `user.username`.

Reply:
Ah, fixed